Sprint-2/implement/tally.js

```diff
@@ -1,3 +1,24 @@
-function tally() {}
+function tally(argument) {
```

Comment:
Every non-arrow function has an implicit array-like object named `arguments` that contains the values of the arguments passed to that function. 
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Naming a parameter/variable `argument` has a higher chance to lead to error.